src/java.base/share/classes/java/lang/String.java

```diff
@@ -4223,19 +4223,29 @@ private static int outdent(List<String> lines) {
      *     <td>continuation</td>
      *     <td>discard</td>
      *   </tr>
+     *   <tr>
+     *     <th scope="row">{@code \u005Cu...uXXXX}</th>
+     *     <td>unicode escape</td>
+     *     <td>single UTF-16 code unit equivalent</td>
+     *   </tr>
      *   </tbody>
      * </table>
      *
-     * @implNote
-     * This method does <em>not</em> translate Unicode escapes such as "{@code \u005cu2022}".
-     * Unicode escapes are translated by the Java compiler when reading input characters and
-     * are not part of the string literal specification.
-     *
      * @throws IllegalArgumentException when an escape sequence is malformed.
      *
-     * @return String with escape sequences translated.
+     * @return String with escape sequences and unicode escapes translated.
+     *
+     * @implNote Normally, unicode escapes are translated by the compiler before string
```

Comment:
A minor comment on the implNote is that it better to drop "Normally," from the beginning of this sentence and "However," from the second sentence. I think it would read a bit better.

Reply:
Changed